inst/include/population_dynamics/population/population.hpp

```diff
@@ -40,15 +40,12 @@ struct Population : public fims_model_object::FIMSObject<Type> {
   size_t nages;         /*!< total number of ages in the population*/
   size_t nfleets;       /*!< total number of fleets in the fishery*/
 
-  // constants
-  const Type proportion_female =
-      0.5; /*!< Sex proportion fixed at 50/50 for M1*/
-
   // parameters are estimated; after initialize in create_model, push_back to
   // parameter list - in information.hpp (same for initial F in fleet)
   fims::Vector<Type>
       log_init_naa;         /*!< estimated parameter: log numbers at age*/
   fims::Vector<Type> log_M; /*!< estimated parameter: log Natural Mortality*/
+  fims::Vector<Type> proportion_female; /*!< estimated parameter: proportion female by year or age */
```

Comment:
Question for documentation: Does this vector of estimated parameters vary by year, or does it only vary by age?

Reply:
fixed - only by age for now